dashboard/app/models/user.rb

```diff
@@ -290,6 +290,12 @@ def User.find_by_email_or_hashed_email(email)
       User.find_by(email: '', hashed_email: User.hash_email(email.downcase))
   end
 
+  def User.find_channel_owner(encrypted_channel_id)
+    owner_storage_id, _ = storage_decrypt_channel_id(encrypted_channel_id)
+    user_id = PEGASUS_DB[:user_storage_ids].where(id: owner_storage_id).first[:user_id]
+    User.find_by_id(user_id)
```

Comment:
The more common way to find by id is `User.find(user_id)` (if you're ok with throwing an exception if the id doesn't exist, which it seems you probably are since the above line has similar behavior).